src/behavior.js

```diff
@@ -66,6 +66,9 @@ const Behavior = MarionetteObject.extend({
   destroy() {
     this.stopListening();
 
+    // Remove behavior-only triggers and events
```

Comment:
A `!this.view._isDestroyed` check here will prevent the call in the most common case (behavior being destroyed together with the view)

Reply:
good catch.  I also noticed this check is already happening inside `_removeBehavior` so I'm going to move the `undelegate`s inside that method as well.  I think this makes sense anyhow as if you're removing the behavior for any reason you'll want these `undelegate`s to occur (if for some reason we reuse `_removeBehavior` in the future)